### DISCUSSION TITLE
How to set "individual shipping"?

### DISCUSSION THREAD

**Discussion by towarisch** [Second-Hand-Friends/kleinanzeigen-bot#363] [Q&A]
Hello, 
how can I select "individueller Versand" in the shipping options via ad.yaml?

**Comment by Heavenfighter** [comment]
~~Its not possible since the only the defined options (DHL,HERMES,..) are possible. It could be done with a new CUSTOM option but this can produce error when downloading ads because the identification can only done with the shipping price. This is the only criteria we have on the page. When the custom shipping price would match one of the predefined prices, it would go wrong.~~

**EDIT**: I have to revert my statement, it should work. You coult leave the shipping_options empty and set only shipping_costs and shipping_type. This should work, as long as the price not match one of the prices from the delivery services as mentioned above.
